source/HailPanel.cpp

```diff
@@ -55,25 +55,18 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
-	else if(gov->IsEnemy())
+	else if(gov->IsEnemy() && !ship->IsDisabled())
```

Comment:
You can see he's disabled in the target reticle, and that I also am in my flagship reticle. While options to request assistance are greyed out, this situation's falldown is probably not acceptable to MZ